cr-synchronizer/main.go

```diff
@@ -1,14 +1,23 @@
 package main
 
 import (
+	"context"
 	"flag"
 	"github.com/netcracker/cr-synchronizer/getters"
+	"os"
+	"strconv"
 )
 
 func main() {
+	ctx := context.Background()
```

Comment:
Can't see handler for OS signals and cancelling context. It will make container more k8s native and rapidly reflect to k8s controlling commands